### PR TITLE
Add FAQ for "import" vs destructuring

### PR DESCRIPTION
**Pull request by hawkrives**
It's an initial draft, but I'm open to feedback. I might come back and edit it myself later.

### CODE REVIEW

**Comment by xtuc** [comment]
I think it's also worth talking about `interopRequire` (https://twitter.com/wSokra/status/828902492893241344)

File: docs/faq.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ description: Frequently Asked Questions and Answers
...
+
+- In CommonJS, imports are copies of exported values.
+    - You can use `const`, `let`, or `var` to define the binding type (`const x = require('foo')`)
+    - The properties of that imported value are mutable (`const foo = require('foo'); foo.thing = 5`) 
+- In ES6, imports are live read-only views on exported values.
+    - That means that unqualified imports (`import x from 'foo'`) are like const-declared variables.
+    - The properties of a module object `foo` (`import * as foo from 'foo'`) are essentially frozen – you cannot modify `foo` itself.
+    - Qualified imports (`import {thing} from foo`) are also like const-declared variables.
+
+Because the exports of a module are live, `import` required different syntax than object destructuring, as it does
```
Comment (existentialism) [inline docs/faq.md]: nit: required -> requires

**Comment by hawkrives** [comment]
Other relevant things to include:

- https://github.com/babel/babel/pull/4964 (spec mode for `transform-es2015-modules-commonjs`)
- https://twitter.com/left_pad/status/829075501134315520 [thread]
    - @ghost: […] I'm not sure what "live" means in this context—that sounds like the opposite of what they are
    - @hzoo: `export let c = 3; export function incCounter() { c++; }` – You can't modify counter directly but it will update
    - @hzoo: I guess we have to make a distinction between read-only and live (the ex in the link pasted should explain it)
    - @ghost: if it helps, as an outsider to this, my reaction was along the lines of "live? as opposed to what?"

**Comment by hawkrives** [comment]
I guess I should come back to this.

It's on my TODO list for this week, now.

**Comment by hzoo** [comment]
This would be great to add, going to close but feel free to reopen again

**Pull request closed without merging**